lib/bunny/exceptions.rb

```diff
@@ -97,6 +97,12 @@ def initialize(frame)
     end
   end
 
+  class ConnectionManuallyClosed < Exception
+    def initialize
+      super('Connection has been manually closed. Call #start to reopen it')
```

Comment:
I'm not sure how I feel about recommending this. It may or may not be safe and I don't think we should support reopening at all. It is hard and is not any easier to use than opening a new connection.
So let's drop the recommendation.